### ISSUE TITLE
Submit limit by form

### ISSUE THREAD

**Issue by almet** [Kinto/formbuilder#159]

**Comment by sahildua2305** [comment]
@almet can you please provide more information about this issue?

**Comment by almet** [comment]
Sure thing: The goal here is to have a maximum number of answers a form can accept. I'm not sure how to implement this, maybe we should add a mechanism on Kinto collections to have a maximum number of items. I would be interested to have @leplatrem input on this.

**Comment by sahildua2305** [comment]
Good idea! So we will need to implement this on Kinto project and then use that feature here, right?
I'd love to work on this once we hear from @leplatrem 😄
